datasets/skab/readme.txt

```diff
@@ -0,0 +1,54 @@
+Datasets description - https://github.com/waico/SKAB 

```

Comment:
Notes to SKAB experiments (GAN-AD-SKAB.ipynb):

- Anomaly scores in plots do not seem to correspond to the predictions (why is `df_anomaly_free_test` provided to `predict_anomaly_scores` call while `df_with_anomalies` is provided to identify_anomaly call?).
- Why are labels generated artificially instead of using labels from SKAB? Without using those labels we cannot compare results to the results from the original paper (https://arxiv.org/pdf/1809.04758.pdf) and thus we cannot easily validate the correctness of the implementation.